### PR TITLE
Get Topiary running on Travis

### PR DESCRIPTION
**Pull request by tavinathanson**
In this PR:

* Copied `mhctools`'s Travis stuff, including installing `netmhc-bundle`
* Install EnsemblRelease 75 within the test that uses it (I don't like needed two lines for this; is there a better way, currently?)
* Re-Travis-encrypted the `mhcbundle` and `pypi` passwords for the `topiary repo
* As discussed remove, all the `remap_novel_genes_onto_ensembl_ids` logic since it's not used/tested (and the linter found issues with it)
* Fixed an issue with `mutant_epitope_predictor` that the linter found; this looks like it warrants a test (see issue #24)

[Review on Reviewable](https://reviewable.io/reviews/hammerlab/topiary/25)

### CODE REVIEW

File: test/test_mutant_epitope_predictor.py [hunk 1]
```diff
@@ -21,6 +21,9 @@
 from topiary import MutantEpitopePredictor, epitopes_to_dataframe
 from varcode import Variant, VariantCollection
 
+ensembl.download()
```
Comment (iskandr) [inline test/test_mutant_epitope_predictor.py]: It's weird to do the download/index here, what do you think of adding `pyensembl install --releas 75` to the travis config?

**Comment by iskandr** [comment]
Other than the question of where an Ensembl release should get installed, LGTM

File: test/test_mutant_epitope_predictor.py (hunk 1 above)
Comment (tavinathanson) [inline test/test_mutant_epitope_predictor.py]: I had that as the first commit, but felt that logic should be coupled with where it's used. When coupled with the test using it, no one needs to "remember" to add some random script.

Comment (iskandr) [inline test/test_mutant_epitope_predictor.py]: Conversely, it gives us less control over where hefty jobs run. Let's keep
it for now as is but possibly revisit if other Ensembl releases become
problematic for Topiary.

Comment (tavinathanson) [inline test/test_mutant_epitope_predictor.py]: Yeah, that's true. I do think we should solve that problem if it comes up vs. beforehand.

**Comment by tavinathanson** [comment]
As discussed offline: I'm changing this to only run on `python2` until we get PyEnsembl running properly on Travis.

**Pull request merged**